logiclearner/main/tests/test_views.py

```diff
@@ -10,3 +11,29 @@ def test_smoketest(self):
         response = self.client.get("/smoketest/")
         self.assertEquals(response.status_code, 200)
         self.assertContains(response, 'PASS')
+
+
+class StatementListAPIViewTest(TestCase):
```

Comment:
For api views, I generally like to add tests for POST/PUT operations to ensure the authorization layer is setup properly.